lib/msal-node/src/cache/NodeCacheManager.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+const fs = require('fs');
```

Comment:
We should use `import` instead of require everywhere by default.

Reply:
Done. Using `fspromises` now.